app/controllers/job_invocations_controller.rb

```diff
@@ -87,7 +88,11 @@ def action_permission
 
   def prepare_composer
     if params[:feature].present?
-      JobInvocationComposer.for_feature(params[:feature], params[:host_ids], {})
+      JobInvocationComposer.for_feature(
+        params[:feature],
+        params[:host_ids],
+        params[:inputs].permit!.to_hash
```

Comment:
this will not work if inputs are not sent, which is the case for REX features which are marked as host_action_button (ran from host detail page from the schedule job drop down button), here's the trace

```
2018-02-23T11:44:23 9991c3b7 [app] [F]   
2018-02-23T11:44:23 9991c3b7 [app] [F] NoMethodError (undefined method `permit!' for nil:NilClass):
2018-02-23T11:44:23 9991c3b7 [app] [F]   
2018-02-23T11:44:23 9991c3b7 [app] [F] /home/ares/Projekty/Zdrojaky/foreman_remote_execution/app/controllers/job_invocations_controller.rb:94:in `prepare_composer'
 | /home/ares/Projekty/Zdrojaky/foreman_remote_execution/app/controllers/job_invocations_controller.rb:34:in `create'
 | app/controllers/concerns/application_shared.rb:15:in `set_timezone'
 | app/models/concerns/foreman/thread_session.rb:32:in `clear_thread'
 | app/controllers/concerns/foreman/controller/topbar_sweeper.rb:12:in `set_topbar_sweeper_controller'
 | lib/middleware/catch_json_parse_errors.rb:8:in `call'
 | lib/middleware/session_safe_logging.rb:17:in `call'
 | lib/middleware/tagged_logging.rb:18:in `call'
```

since tests were green, I'd like to ask also for adding a tests with no inputs